### DISCUSSION TITLE
Anybody using PerspectiveWidget on Google Colab ?

### DISCUSSION THREAD

**Discussion by jpv** [perspective-dev/perspective#2424] [General]
No output at all with something like 

`
from google.colab import output
output.enable_custom_widget_manager()

from perspective import PerspectiveWidget, Table
PerspectiveWidget(df.collect().to_pandas(), group_by=["type","arrivalDate"])
`

**Comment by timkpaine** [comment]
Colab is a proprietary product, best bet is to ask google to support it: https://github.com/cytoscape/ipycytoscape/issues/148#issuecomment-683344889

@blois
